arangodb-net-standard.Test/CollectionApi/CollectionApiClientTest.cs

```diff
@@ -103,5 +109,28 @@ public async Task PostCollectionAsync_ShouldThrow_WhenCollectionNameInvalid()
             });
             Assert.Equal(1208, ex.ApiError.ErrorNum);
         }
+
+        [Fact]
+        public async Task GetCollectionCountAsync_ShouldSucceed()
+        {
+            var newDoc = await _documentApi.PostDocumentAsync(_fixture.TestCollection, new PostDocumentsOptions());
+            var response = await _collectionApi.GetCollectionCountAsync(new GetCollectionCountOptions
+            {
+                CollectionName = _fixture.TestCollection
+            });
+
+            Assert.Equal(HttpStatusCode.OK, response.Code);
```

Comment:
I think we should check the response object.  In the implementation I see we don't use the camelCase property conversion options, so it seems like deserialisation might not work?  Of course maybe it does work and I'm wrong. Either way the test should ensure it by checking expected values for the different properties.